test-app/app/components/docs/components/phone-input/all-options.hbs

```diff
@@ -0,0 +1,129 @@
+<h2>Basic Options</h2>
```

Comment:
Where can I see the rendered output of this?

Reply:
That's the docs page that can be found on localhost 4202 locally or on github pages: https://qonto.github.io/ember-phone-input/versions/master/